webknossos-tracingstore/proto/SkeletonTracing.proto

```diff
@@ -48,6 +48,7 @@ message Tree {
      required string name = 7;
      required int64 createdTimestamp = 8;
      optional int32 groupId = 9;
+     optional bool isVisible = 10;
```

Comment:
For the record, isVisible will be undefined in that case, and the frontend reducer will take care of setting that to true. I made the isVisible attribute in the server tracing type optional, as well, so that we are forced to do the "casting".